tools/mason/MasonModify.chpl

```diff
@@ -91,6 +94,20 @@ proc modifyToml(add: bool, spec: string, external: bool, system: bool,
         newToml = masonExternalAdd(toml, dependency, spec);
       }
       else {
+
+        if !skipCheck {
+          // ensure that dependency exists and check package type
+          const depToml = getDepToml(dependency, version);
+
+          // Ensure path exists to maintain compatibility with old
+          // versions of TOML files which did not require the type tag.
+          if depToml.pathExists("brick.type") {
+            if depToml["brick.type"]!.s != "library" {
```

Comment:
So this will still behave as before the addition of the `type` key? 
e.g., a `.toml` file without a `type` can be added as a dependency, but if a `type` key is specified, it must have a value of `library`?

Reply:
yes, that is correct.